src/textbooks/textbook-form/TextbookForm.scss

```diff
@@ -9,6 +9,10 @@
   margin-bottom: map-get($spacers, 4);
   border-radius: .5rem;
 
+  .form-control::placeholder {
+    font-size: .875rem;
```

Comment:
```suggestion
    font-size: $font-size-sm;
```

Reply:
This rule is unnecessary. I have removed it.